### ISSUE TITLE
Newbie a little puzzled with the init choices

### ISSUE THREAD

**Issue by jlengrand** [open-wc/open-wc#370]
Heyo, 

First of all, you guys are making an amazing job. It's insane to see how much has been pulled off since the last time I checked the project couple months ago. 

 I do have a remark. When I run `npm init` for the first time, it is not obvious to me what the different choices are.

```
[jlengrand@juliens-MacBook-Pro:~/IdeaProjects]$ npm init @open-wc
npx: installed 27 in 1.326s
What would you like to do today?
  > Scaffold a new project
What would you like to scaffold?
Note: Content will be written in a new folder.
    Barebone App
    Starter App
  > Enterprise App (if you feel lost use the Starter App first)
    Lit Element Web Component
    Mono Repo for web components
```

Even having worked with open-wc a little bit in the past, I am not sure what the difference is for example between the barebone app and the Lit Element Web Component. 

I've been browsing around the website, but could not find a place where those are described in an obvious manner. 

I might have missed it, but I'd appreciate something like 

* Starter app : perfect if you have a simple side-project, ......
* Lit element Web Component : Use this if you want to create a single component. No router will be added, ....

Basically, when do I choose what?

Great job folks, keep it up :).

**Comment by thepassle** [comment]
The purpose of each individual generator is described here: https://open-wc.org/developing/create.html#usage

But I do agree we could improve on the experience of the generator a little bit. (e.g.: there's also an option for Enterprise App that doesn't even exist yet). So there's some work to be done there. I can take a look when I come back from my vacation, or anybody else can pick this up and make a PR if they want :-)

Maybe we can even add a link to that page, actually. Something like: "Want to learn more about what the generators do? Visit open-wc.org/developing/create" or something.

**Comment by jlengrand** [comment]
Hey that looks exactly like what I was searching for. 

Adding a link to the command line tool to help people would be enough I feel. I'll check how fast I'm done with my stuff today and come back to it if I have time. No promises though <3.

Enjoy Twitch Conf!

**Comment by jlengrand** [comment]
Alright, just spent some time trying to implement this. However, I can't run the tests locally (getting hundreds of errors about missing files in packages 

```
ERROR in ./packages/testing-helpers/test/lit-html.test.js
Module not found: Error: Can't resolve '@bundled-es-modules/chai' in '/Users/jlengrand/IdeaProjects/open-wc/packages/testing-helpers/test'
```

is a small example. 

Given that I cannot run the tests, nor have any idea how to locally run the altered cli I do not feel confident implementing it, no matter how simple it is.

**Comment by daKmoR** [comment]
did you install with `npm i`? we require `yarn install` as we use `yarn workspaces` in combination with lerna...

oh now that I see it - we definitely need to update the contribution guide 🙈

**Comment by jlengrand** [comment]
Yeah I've ben fighting quite a bit with your setup. 

Now still blocked on multiple dependencies that are required in global scope (mocha, eslint, ....) and that prevent me from committing :P. 
If I ever get past through your pre-commit hooks I might be able to send a PR :)

**Comment by jlengrand** [comment]
You are really not making my work easy :).

<img width="449" alt="Screen Shot 2019-04-13 at 3 06 39 PM" src="https://user-images.githubusercontent.com/921666/56080177-d600da80-5dfd-11e9-96de-2254d2f9514e.png">

**Comment by daKmoR** [comment]
that should be all you need

```bash
git clone git@github.com:open-wc/open-wc.git
cd open-wc
yarn install
yarn test
```
to get up and running... 😬 

hehe yeah... unfortunately a necessary evil 😅

**Comment by LarsDenBakker** [comment]
We need lots of tooling to give people a (as much as possible) tooling free dev setup. It's quite ironic.

**Comment by jlengrand** [comment]
Looking into it @daKmoR .

**Comment by jlengrand** [comment]
Alright, you are correct @daKmoR . I had an old version of `yarn` and used npm install instead to save some time. Looks like I didn't in the end. 

Everything works fine using yarn.

**Comment by jlengrand** [comment]
Fixed by https://github.com/open-wc/open-wc/pull/375

**Comment by daKmoR** [comment]
fixed with the new generator flow 💪 

it's still not 100% clear as with the feature options now there is not much difference between starter-app / barebone-app / web component... would be nice to converge and bring needed options just to the feature selection... => but that's a task for another time

**Comment by jlengrand** [comment]
Hum well if it's not 100% clear then you have to explain me how the issue
is fixed 😂

**Comment by thepassle** [comment]
iirc we added a link to the docs in the generator right? People can find more info about individual generators there

**Comment by jlengrand** [comment]
Alright I know this is closed already but I finally took the time to update and check and yes, it is pretty obvious now what is what. 

Great job!
